x-pack/plugins/security_solution/public/management/cypress/support/saml_authentication.ts

```diff
@@ -34,15 +35,15 @@ export const samlAuthentication = async (
       role: string | SecurityRoleName
     ): Promise<{ cookie: string; username: string; password: string }> => {
       // If config.env.PROXY_ORG is set, it means that proxy service is used to create projects. Define the proxy org filename to override the roles.
-      const rolesFilename = config.env.PROXY_ORG ? `${config.env.PROXY_ORG}.json` : undefined;
-      const sessionManager = new SamlSessionManager(
-        {
-          hostOptions,
-          log,
-          isCloud: config.env.CLOUD_SERVERLESS,
-        },
-        rolesFilename
-      );
+      const rolesFilename = config.env.PROXY_ORG
+        ? `${config.env.PROXY_ORG}.json`
+        : 'role_users.json';
+      const sessionManager = new SamlSessionManager({
+        hostOptions,
+        log,
+        isCloud: config.env.CLOUD_SERVERLESS,
+        cloudUsersFilePath: `${REPO_ROOT}/.ftr/${rolesFilename}`,
```

Comment:
I think our serverless tests (in CI - non-MKI) have been running with these roles: 

https://github.com/elastic/kibana/blob/45a28d40e3e6e0ea825f8ccfacb3ce4ac392a0fb/x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/serverless/es_serverless_resources/README.md

Should that be used here instead of this file path you defined?

If so, you can import `ES_RESOURCES` from `x-pack/plugins/security_solution/scripts/endpoint/common/roles_users/serverless` and use `ES_RESOURCES.roles` here

Reply:
I had another look and `rolesFilename` is actually storing credentials, not the actual roles.
`cloudUsersFilePath` takes the absolute path to the json file with credentials by role, `roles.yml` is read from `@kbn/es`

Reply:
cf2bc2d6e30d3fcd7ec700ea71c45ee288acc190